src/components/UsersList.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+import User from "./User";
+import spinner from "../images/spinner.gif";
+
+function UsersList({ users, isLoading }) {
+  const [user, setUser] = useState(null);
+  const handleClick = (e) => {
+    setUser(users[e.target.value]);
+  };
+  useEffect(() => {
+    setUser(users[0]);
+  }, [users]);
+  return (
+    <>
+      {isLoading && <img src={spinner} alt="loading" />}
+      {users.map((userLink, i) => (
+        <li className="userLink" onClick={handleClick} key={i} value={i}>
+          {userLink.name.first} {userLink.name.last}
+        </li>
+      ))}
```

Comment:
`value` is not the the value here but the `index`;
`userLink` is not the user link here but the `user`;
`key` could be not unique of you use the index;
```suggestion
      {users.map((user, index) => (
        <li className="userLink" onClick={handleClick} key={user.login. uuid} index={index}>
          {user.name.first} {user.name.last}
        </li>
      ))}
```